services/movie_session.py

```diff
@@ -42,3 +42,13 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(
+    movie_session_id: int
+) -> list[dict]:
+    tickets = get_movie_session_by_id(movie_session_id).tickets.all()
```

Comment:
In this case you do two queries. Simplify code and make only one query to database